src/util/deleteThought.js

```diff
@@ -89,7 +89,7 @@ export const deleteThought = () => {
       store.dispatch(cursorBack())
     }
     else {
-      store.dispatch({ type: 'setCursor', thoughtsRanked, editing: state.editing, offset })
+      store.dispatch({ type: 'setCursor', thoughtsRanked: prev ? thoughtsRanked : [thoughtsRanked[0]], editing: state.editing, offset })
```

Comment:
This doesn't work with the cursor is more than one thought.

e.g.

- a
  - b
    - =test
    - c [cursor]

After `deleteThought`, the cursor will incorrectly jump up to the top level instead of moving up one level.

You can't use `[thoughtsRanked[0]]` as it throws away all of the other thoughts in the context. To get the context of a given context (i.e. one level up), try the `contextOf` function.

Reply:
I'll take a look into it.